chapters/en/Unit 1 - Fundamentals/imaging.mdx

```diff
@@ -0,0 +1,29 @@
+In digital imaging, sensor technology is pivotal. Single sensing elements, such as photodiodes, are basic forms that require movement along both x and y axes to form a two-dimensional image. In contrast, sensor strips, which are more common, capture images linearly in one direction. To obtain a complete 2D image, these strips necessitate motion in the perpendicular direction. Such technology is commonly found in devices like flatbed scanners and is also utilized in airborne imaging systems. In more specialized applications, like medical imaging (e.g., CAT scans), ring-configured sensor strips are used. These setups involve complex reconstruction algorithms to transform the captured data into meaningful images, highlighting the advanced computation involved in modern imaging technologies. Sensor arrays, like CCDs in digital cameras, consist of 2D arrays of sensing elements. They capture a complete image without requiring motion, as each element detects part of the scene. These arrays are advantageous as they don't require movement to capture an image, unlike single sensing elements and sensor strips. The captured energy is focused onto the sensor array, converted into an analog signal, and then digitized to form a digital image.
+
+3. Digital Image Formation and Representation
+The core of digital image formation is the function f(x, y), which is determined by the illumination source i(x, y), and the reflectance r(x, y) from the scene. In transmission-based imaging, such as X-rays, transmissivity takes the place of reflectivity. The digital representation of an image is essentially a matrix or array of numerical values, each corresponding to a pixel. The process of transforming continuous image data into a digital format is twofold: sampling, which digitizes the coordinate values, and quantization, which converts amplitude values into discrete quantities. The resolution and quality of a digital image are significantly dependent on the number of samples and discrete intensity levels used. The dynamic range of the imaging system, which is the ratio of the maximum measurable intensity to the minimum detectable intensity, also plays a crucial role in the appearance and contrast of the image.
+
+4. Understanding Resolution in Digital Imaging
+Spatial resolution refers to the smallest distinguishable detail in an image and is often measured in line pairs per unit distance or pixels per unit distance. The meaningfulness of spatial resolution is context-dependent, varying according to the spatial units used. For example, a 20-megapixel camera typically offers higher detail resolution than an 8-megapixel camera. Intensity resolution relates to the smallest detectable change in intensity level and is often limited by the hardware's capabilities. It's quantized in binary increments, such as 8 bits or 256 levels. The perception of these intensity changes is influenced by various factors, including noise, saturation, and the capabilities of human vision.
+

```

Comment:
This is a bit of a rough transition. 

I would add something like, "now that we have see how image are acquired, we can discuss some operation that can be done in it".